packages/phone-number-privacy/src/web3/contracts.ts

```diff
@@ -0,0 +1,10 @@
+import { newKitFromWeb3 } from '@celo/contractkit'
+import Web3 from 'web3'
+import config from '../config'
+
+const web3: Web3 = new Web3(new Web3.providers.HttpProvider(config.blockchain.provider))
```

Comment:
Given that we don't explicitly declare web3 as a dep of this project, it'd be better to let CK handle the interactions with web. Consider using `newKit` instead of `newKitFromWeb3`